components/sup/src/manager/mod.rs

```diff
@@ -221,7 +221,6 @@ pub struct Manager {
     pub state: Arc<ManagerState>,
     butterfly: butterfly::Server,
     census_ring: CensusRing,
-    events_group: Option<ServiceGroup>,
```

Comment:
We'll want to rebase before merging, so we get these eventsrv-related changes naturally.

Reply:
Done